standard/table.lua

```diff
@@ -67,16 +67,10 @@ end
 ---@param tbl table?
 ---@return boolean
 function Table.isEmpty(tbl)
-	if tbl == nil then
+	if not tbl then
 		return true
 	end
-	-- luacheck: push ignore
-	--it is intended that the loop is executed at most once
-	for _, _ in pairs(tbl) do
-		return false
-	end
-	-- luacheck: pop
-	return true
+	return not next(tbl) and true or false
```

Comment:
Wouldn't 
```suggestion
	return not next(tbl)
```
be enough?

Reply:
wouldn't return a boolean

![Screenshot 2022-09-27 13 57 57](https://user-images.githubusercontent.com/75081997/192519226-e760c843-865a-4e9b-9e2c-cfdb20552ced.png)

Reply:
nvm i am stupid, the not makes it bool
so yeah would work

Reply:
`not` always returns a boolean